pkgs/tools/audio/bark/default.nix

```
@@ -1,22 +1,45 @@
-{ lib
-, rustPlatform
-, fetchFromGitHub
-, pkg-config
-, alsa-lib
-, speexdsp
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  alsa-lib,
+  cmake,
+  libopus,
+  soxr,
 }:
-rustPlatform.buildRustPackage {
-  pname = "bark";
-  version = "unstable-2023-08-22";
+let
+  version = "0.6.0";
   src = fetchFromGitHub {
     owner = "haileys";
     repo = "bark";
-    rev = "2586b9fb58b496f8ef06f516c9cd3aace77521f7";
-    hash = "sha256-sGroae6uJhB9UIpFmvt520Zs9k0ir7H8pGkhKJmVWek=";
+    rev = "v${version}";
+    hash = "sha256-JaUIWGCYhasM0DgqL+DiG2rE1OWVg/N66my/4RWDN1E=";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "bark";
+  inherit version src;
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

This is an IFD, you'll have to copy the Cargo.lock locally and then write

```suggestion
    lockFile = ./Cargo.lock;
```